mu4/instruments/qml/MuseScore/Instruments/InstrumentsDialog.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.9
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Instruments 1.0
+
+QmlDialog {
+    id: root
+
+    property string canSelectMultipleInstruments: "true"
```

Comment:
why this property is string, not bool?

Reply:
This property can be configured through URI

Reply:
Added possible pass bool by uri

Reply:
thanks!!!
fixed